main.js

```diff
@@ -0,0 +1,108 @@
+import './style/style.sass'
+import './style/reset.css'
+// import $ from 'jquery'
+import { v4 as uuidv4 } from 'uuid'
+
+const { JSDOM } = require( "jsdom" );
```

Comment:
your test code should affect regular code, so jsdom and require of jquery shouldn't be here